src/sif/dynamiccost.cc

```diff
@@ -260,6 +260,13 @@ void DynamicCost::RelaxHierarchyLimits(const float factor, const float expansion
   }
 }
 
+void DynamicCost::RelaxHierarchyLimits(const bool using_bidirectional) {
```

Comment:
maybe we should make the other version of this method with the two floats, private, so that we dont make this mistake again, though im not sure there arent other places that call this

Reply:
or just remove it completely keeping single:
```cpp

void DynamicCost::RelaxHierarchyLimits(const bool using_bidirectional) {
  // since bidirectional A* does about half the expansion we can do half the relaxation here
  const float relax_factor = using_bidirectional ? 8.f : 16.f;
  const float expansion_within_factor = using_bidirectional ? 2.0f : 4.0f;
  
  for (auto& hierarchy : hierarchy_limits_) {
    hierarchy.Relax(factor, expansion_within_factor);
  }
}
```

What you say?

Reply:
ship it

Reply:
i like your suggestion of combining the two into one with just the single boolean parameter. ill wait for you to make that change before putting a ship on it

Reply:
I've just pushed the updating commit